library/src/lib/calendar/calendar2/calendar2-views/calendar2-month-view/calendar2-month-view.component.html

```diff
@@ -1,3 +1,22 @@
-<p>
-  calendar2-month-view works!
-</p>
+<div class="fd-calendar__months">
+    <ul class="fd-calendar__list">
+        <li
+            class="fd-calendar__item"
+            *ngFor="let month of monthNames; let i = index"
+            [ngClass]="
+                (i + 1 === currentMonth ? ' fd-calendar__item--current' : '') +
```

Comment:
Avoid magic numbers

also, the better syntax for things like this would be 
`<some-element [ngClass]="{'firstclass': true, 'secondclass': true, 'thirdclass': false}">...</some-element>`

Reply:
Agreed for the better syntax, will change it. Not sure what you mean with `magic numbers`. SInce we agreed on working with 1-12 months I'm not sure how to handle the indexing of the ngFor (0-11) without adding some additional logic in the component.

Reply:
You can just define a const on top of the component with value 1 which is called something like `loopMonthOffset` and also documented. It's just for clarity 🤷‍♀